Tests/Kernel/TestKernelFilePermissions.cpp

```diff
@@ -82,7 +82,7 @@ TEST_CASE(test_change_file_location)
     EXPECT(fchmod(fd, 06755) != -1);
 
     auto suid_path_or_error = FileSystem::read_link(DeprecatedString::formatted("/proc/{}/fd/{}", getpid(), fd));
-    EXPECT(!suid_path_or_error.is_error());
+    MUST(suid_path_or_error);
 
     auto suid_path = suid_path_or_error.release_value().to_deprecated_string();
```

Comment:
```
auto suid_path = MUST(FileSystem::read_link(DeprecatedString::formatted("/proc/{}/fd/{}", getpid(), fd));).to_deprecated_string()
```

instead of all the stuff above.

(In general, prefer smaller, high-quality changes over huge rewrites with issues like this, maybe?)

Reply:
Yeah, you're right. I thought it would really be a simple search-and-replace, if I had known in advance that each callsite needs individual treatment I would have approached this differently.